src/app/features/search/search.component.ts

```diff
@@ -22,7 +32,7 @@ export class SearchComponent {
 
     searchTerm = searchTerm.trim();
 
-    if (searchTerm) {
+    if (searchTerm || searchTerm !== '') {
```

Comment:
Isn't an empty string falsy? Do you need the second check

Reply:
This is leftover code, thank you